### DISCUSSION TITLE
Copy/Pasted yaml to dashboard and it doesn't work.

### DISCUSSION THREAD

**Discussion by jsomontan** [uptrace/uptrace#405] [General]
Hi,

   I copied and pasted the yaml onto the Uptrace console and it didn't do anything.  I clicked on "create dashboard from yaml", "pasted the yaml contents" and it didn't create the dashboard.  Not sure what I'm doing wroing.

Thanks,

+ Joe

config/dashboard-templates/uptrace.aws.10.ec2_instances.yml

**Comment by vmihailenco** [comment]
Hi,

That particular template had a type which should be fixed by this commit: https://github.com/uptrace/uptrace/commit/043098eb692b3cea1d940dae5776b5c7f8225ed9
